neuralmonkey/decoders/autoregressive.py

```diff
@@ -77,20 +90,25 @@ class DecoderFeedables(NamedTuple(
         "DecoderFeedables",
         [("step", tf.Tensor),
          ("finished", tf.Tensor),
-         ("input_symbol", tf.Tensor),
-         ("prev_logits", tf.Tensor)])):
+         ("embedded_input", tf.Tensor),
+         ("other", Any)])):
     """The input of a single step of an autoregressive decoder.
 
+    The decoder should be able to decode in each step only using this.
```

Comment:
napsal bych "should be able to generate an output symbol only using information contained in this structure."